README.md

```diff
@@ -22,6 +22,15 @@ Project is licensed under [Apache License 2.0](http://www.apache.org/licenses/LI
 
 -----
 
+## Release notes
+
+Release notes are available in [release-notes](./release-notes/).
```

Comment:
I think this should first link to Jackson release page: https://github.com/FasterXML/jackson/wiki/Jackson-Releases (which further links to specific release versions like https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.20). Maintaining separate per repo pages is additional work that probably won't happen.

However! For specific case of 2.20 I agree that something should be added here, like section below.

Reply:
Fine with me.

I was actually a bit confused as there are information in multiple places: the wiki in the "top" repo, README in the "top" repo, wiki in this repo, release note markdown in this repo, README in this repo.

(By the way, we can't make PRs on the wiki pages, it's a bit of a drawback of using the wiki vs. plain markdown files (and optionally generate a static website from it))